spotify/base.py

```diff
@@ -53,3 +55,36 @@ def pull_library_data(self, user):
         self.track_utils.add_track_artist_m2m(track_dicts)
         self.library_utils.add_library_to_db(track_dicts, user)
         return
+
+    def send_post(self, url, data):
```

Comment:
This will likely need a `# pragma: no cover`. You will figure that out once you get tests passing. It'll also require mocking this out in tests.